README.md

```diff
@@ -18,7 +18,7 @@
 	* [Restart containers when a certificate is renewed](#restart-containers-when-a-certificate-is-renewed)
 	* [Call a reload command on containers when a certificate is renewed](#call-a-reload-command-on-containers-when-a-certificate-is-renewed)
 	* [Run a custom deploy hook script](#run-a-custom-deploy-hook-script)
-	* [Run a service in a cluster environment](#run-a-service-in-a-cluster-environment)
+	* [Run the container in a cluster environment](#run-the-container-in-a-cluster-environment)
```

Comment:
Nice catch !

Reply:
I think it should be changed to "Running as a service in a cluster environment"

Reply:
Well, I agree with the mention `container` instead of `service`. The point of this section is to make my container aware of the fact that the targets for autoreload are services, not single containers. But the fact that `letsencrypt-dns` is also a pod or a service is out of this section purpose. It may or may be not, but here we are still talking about the configuration of the container itself.